### PR TITLE
mobile version updated

### PR DESCRIPTION
**Pull request by dhy-612**
Milestone 2 
Completed a basic structure of the mobile version of the portfolio

Two Sections I have created:
The Toolbar (or header).
The headline section (right after the header).
I have used Flexbox in all 2 sections.
I have made the basic structure of the project i.e. Header, Headline using Figma templates.
I have created a work section.
I have made the "About myself" section (placed after the projects and before the contact form)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -78,11 +79,13 @@ header {
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
-  order: 1;
-  flex-grow: 0;
+
+  /* order: 1;
+  flex-grow: 0; */
```
Comment (NahnahAJ) [inline styles.css:84]: - Kindly avoid leaving commented pieces of code in your project. It is unprofessional and makes your code unclean. :+1:

File: styles.css [hunk 2]
```diff
@@ -66,9 +66,10 @@ header {
   display: flex;
   align-items: center;
   color: #172b4d;
-  flex: none;
+
+  /* flex: none;
   order: 0;
```
Comment (NahnahAJ) [inline styles.css:71]: - Kindly avoid leaving commented pieces of code in your project. It is unprofessional and makes your code unclean. :+1:

File: styles.css [hunk 3]
```diff
@@ -99,3 +102,303 @@ header {
...
+
+.About-Top {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+
+  /* width: 327px;
+  height: 409px; */
```
Comment (NahnahAJ) [inline styles.css:302]: - Kindly avoid leaving commented pieces of code in your project. It is unprofessional and makes your code unclean. :+1:

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
Hi @dhy-612 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good commit messages :heavy_check_mark: 
- Professional README :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Writing a more detailed pull request next time :+1: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (Yerimah) [reply styles.css:71]: - I would encourage you to always implement changes made by a reviewer before requesting another review. Kindly make these changes before submitting for review. 🙏

**Review by Yerimah — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES 🔴 

Hi @dhy-612, 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4b70556d45436b5038554458344c37536b552f67697068792e676966

## Highlights
- No linter error ✔️ 
- GitHub flow used ✔️ 
- Descriptive PR title and summary ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by NahnahAJ — APPROVED** [review]
Hi @dhy-612 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good use of GitHub flow :heavy_check_mark: 
- Good commit messages :heavy_check_mark:
- Professional README :heavy_check_mark:
- No linter errors :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**